test/fixtures/output/connect/plugin/install/remote-bad-md5.golden

```diff
@@ -0,0 +1,10 @@
+Beginning installation.
+
+Using the only available Confluent Platform installation at "test/fixtures/input/connect/confluent-7.3.3".
+Do you want to install this plugin into "test/fixtures/input/connect/confluent-7.3.3/share/confluent-hub-components"? (y/n): 
+License:
+Example License 1.0
+url-to-license-goes-here
```

Comment:
Can we fit this on one line? Also, it would be great to use an actual URL so we can better visualize what this will look like in practice.
```suggestion
License: Example License 1.0 (url-to-license-goes-here)
```